app/assets/javascripts/app/views/generic/searchable_multiple_select_item.jst.eco

```diff
@@ -0,0 +1,4 @@
+<div>
+    <span class='selected_item' onclick="$(this).closest('div').remove()"><%= @title %></span>
```

Comment:
Please don't use inline JS. Move it to the controller that handles the template.